CMakeLists.txt

```diff
@@ -262,6 +262,26 @@ endif()
 #set(SETUP_PY ${CMAKE_CURRENT_SOURCE_DIR}/bindings/python/setup.py)
 #configure_file(python/setup.py.in ${SETUP_PY} @ONLY)
 
+# support creating some basic binpkgs via `make package`
+set(LIBIIO_VERSION_PATCH ${LIBIIO_VERSION_GIT})
```

Comment:
Use g${LIBIIO_VERSION_GIT}
With the small 'g' before, it's the convention when git tags are used in versions.
Also instead of defining LIBIIO_VERSION_PATCH you could just set that line 277.